config/routes.rb

```diff
@@ -173,6 +173,10 @@ def show_step(name)
         edit_step :are_clients_outgoings_more_than_income, alias: :outgoings_more_than_income
       end
 
+      namespace :capital, constraints: -> (_) { FeatureFlags.means_journey.enabled? } do
+        edit_step :which_savings, alias: :add_saving
```

Comment:
Open to any naming suggestions!

Reply:
@hiboabd As per the [Sheet](https://docs.google.com/spreadsheets/d/1gdDpuyqWO6MJgUT1SOy0nbHd8AAdp1YebBzh1WDG040/edit#gid=0) -> 155 the url should be '/which_savings_does_client_have'

Reply:
Ah perfect! Thanks i've corrected the url @ripan